docs/DeveloperGuide.md

```diff
@@ -1,29 +1,204 @@
 # Developer Guide
 
 ## Acknowledgements
+Credits to [Personbook](https://github.com/nus-cs2113-AY2223S2/personbook)
+for some reused skeleton code and inspiration on OOP implementation.
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Setting up
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Design 
+### Architecture
+![Architecture_Diagram](uml/images/DinerDirectorArchitectureDiagram.png)
```

Comment:
Imo this image is a bit too big, you could reduce its size. Currently, it spans twice the height of my computer screen